src/main/scala/io/iohk/ethereum/consensus/ConsensusConfig.scala

```diff
@@ -32,6 +33,7 @@ object ConsensusConfig extends Logger {
     final val HeaderExtraData = "header-extra-data"
     final val BlockCacheSize = "block-cashe-size"
     final val MiningEnabled = "mining-enabled"
+    final val OptOut = "opt-out"
```

Comment:
I am wondering if `opt-out` is not too general. Maybe `treasury-opt-out` or sth like that?

Reply:
For the configuration only or do you mean for the header field as well? I was in doubt whether renaming it to treasuryOptOut or not

Reply:
I would prefer to change it in both places. It will be more consistent